tests/lang/test_amharic.py

```diff
@@ -10,4 +10,5 @@
 def test_am_sbd(am_default_fixture, text, expected_sents):
     """Amharic language SBD tests"""
     segments = am_default_fixture.segment(text)
+    segments = [s.strip() for s in segments]
```

Comment:
Why do the sentences now possibly have extra whitespace?

Reply:
That's to make it nondestructive.

Example:
```python
import pysbd
text = "My name is Jonas E. Smith. Please turn to page no. 55."
seg = pysbd.Segmenter(language="en", clean=False, char_span=False)
segments = seg.segment(text)
# ['My name is Jonas E. Smith. ', 'Please turn to page no. 55.']
"".join(segments) 
# "My name is Jonas E. Smith. Please turn to page no. 55."
```
Earlier it was without trailing spaces. This is in a way important but anyway with spacy start_char index is imporant
`['My name is Jonas E. Smith.', 'Please turn to page no. 55.']`

Reply:
Oh, perfect 👌